.builds/linux.yml

```diff
@@ -23,3 +23,7 @@ tasks:
       $HOME/.cargo/bin/rustup toolchain install --profile minimal $msrv
       rm Cargo.lock
       $HOME/.cargo/bin/cargo +$msrv test
+  - rustdoc: |
+      $HOME/.cargo/bin/rustup toolchain install nightly -c rust-docs
+      cd vte
+      RUSTDOCFLAGS="--cfg docsrs -Dwarnings" $HOME/.cargo/bin/cargo +nightly doc --features=ansi --no-deps
```

Comment:
Is there any reason to have the `--cfg docsrs` here? What exactly does it achieve? I'd assume rustdoc verification should be largely independent of target documentation platform.

Reply:
I am just following:
https://users.rust-lang.org/t/how-to-document-optional-features-in-api-docs/64577/3

 You gave me a link to rand crate:
 >  Any reason not to use doc_auto_cfg instead? (https://github.com/rust-random/rand/blob/master/src/lib.rs#L56)
 
 And it seems that they do the same:
 https://github.com/rust-random/rand/blob/7808f4e1f7d9578e3c08405b1f4e575720d58a51/.github/workflows/test.yml#L37
!

Reply:
>I'd assume rustdoc verification should be largely independent of target documentation platform.

unless you have doc stuff behind `docsrs` feature, it shouldn't really matter.

Reply:
I am almost sure that we can use any name instead of `docsrs`.
Because, it is used only to activate the nightly `doc_auto_cfg` feature.